scss/root/_pagination.scss

```diff
@@ -0,0 +1,37 @@
+@import '../core/variables';
+
+.pager {
+  padding-top: 1.875rem;
+
+  .pager__items {
```

Comment:
There's no reason to nest these selectors. Doing so adds to the compiled stylesheet's bloat.